src/app/visit/visit.component.ts

```diff
@@ -113,4 +120,12 @@ export class VisitComponent implements OnInit {
     ): object is Patient {
         return Object.hasOwn(object, 'name');
     }
+    checkVisit(): void {
```

Comment:
maybe better name for the function would be `createVisitPointsTable` ? or something like this? `checkVisit` doesn't describe much I would change the name.

Reply:
done